torch/_inductor/graph.py

```diff
@@ -1006,6 +1000,17 @@ def init_wrapper_code(self):
         )
         only_cpu = len(device_types) == 0
         device_type = "cpu" if only_cpu else device_types.pop()
+
+        self.device_ops = get_device_op_overrides(device_type)
+
+        self.cuda = "cuda" in self.device_types
+        if self.cpp_wrapper:
+            self.validate_can_generate_cpp_wrapper()
+            self.wrapper_code = (
+                CudaWrapperCodeGen() if self.cuda else CppWrapperCodeGen()
+            )
+            return
+
```

Comment:
> May I know why we need this change?
Here I set the `self.device_ops` before all wrapper_code initilization including `CppWrapperCodeGen`. In case `self.device_ops` being extended and used in `CppWrapperCodeGen` in the future.

Reply:
The code block is for `CppWrapper`. In terms of this PR, we have not supported `CppWrapper` yet.